python/cudf/setup.py

```diff
@@ -40,6 +40,7 @@
         "hypothesis" "mimesis",
         "pyorc",
         "msgpack",
+        "transformers",
```

Comment:
I think this needs to be added to the `requirements.txt` file as well

Reply:
Added it with commit https://github.com/rapidsai/cudf/pull/7942/commits/23e9fa437f0ac088ad4b22960d3fcb75bfd0709c